modules/broker-core/src/main/java/org/wso2/broker/core/transaction/BrokerTransaction.java

```diff
@@ -76,4 +76,9 @@
      * @return transaction supported or not
      */
     boolean isTransactional();
+
+    /**
+     * Actions to be performed on a channel close
+     */
+    void onChannelClose();
```

Comment:
Should we expose the concept of channel to broker-core? Better of we can remove the word channel from this method.

Reply:
:+1: